packages/js-sdk/src/Framework.js

```diff
@@ -7,6 +7,8 @@ const {batchCall} = require("./batchCall");
 const ConstantFlowAgreementV1Helper = require("./ConstantFlowAgreementV1Helper");
 const InstantDistributionAgreementV1Helper = require("./InstantDistributionAgreementV1Helper");
 const fetch = require("node-fetch");
+// yes, this is dirty. truffle is deprecated, so this won't stick forever.
+const truffleConfig = require("../../ethereum-contracts/truffle-config");
```

Comment:
:cold_sweat:

Reply:
if you can't get over it, I can also keep a local diff of this hack instead of pushing it.

Reply:
no, it's fine, if you couldn't find a workaround with less copy pasted code, then let's incur another technical debt here.

Reply:
I think of it as tech debt we know is about to be written off